src/components/Avatar/Avatar.tsx

```diff
@@ -108,17 +114,17 @@ const Avatar: React.FC<AvatarProps> = props => {
       {name && (
         <Typography
           align='center'
-          variant={type === AvatarType.REGULAR ? 'h4' : 'h5'}>
-          <Box fontWeight={theme.typography.fontWeightBold} lineHeight='1.2'>
-            {name}
-          </Box>
+          component={'h4'}
+          variant={type === AvatarType.REGULAR ? 'body1' : 'h5'}>
```

Comment:
Can we make it as a separate const:
```suggestion
          const type = type === AvatarType.REGULAR ? 'body1' : 'h5';
          ...
          <Typography variant={type}>
```

Reply:
> Can we make it as a separate const:

Let's leave it as it is. I'm gonna address this here **[Simplify Avatar component](https://digitalrig.atlassian.net/browse/BUD-208)**